sunpy/visualization/animator/base.py

```diff
@@ -510,20 +510,16 @@ def _sanitize_axis_ranges(self, axis_ranges, data_shape):
             raise ValueError("Length of axis_ranges must equal number of axes")
 
         # Define error message for incompatible axis_range input.
-        incompatible_axis_ranges_error_message = lambda j: \
-          "Unrecognized format for {0}th entry in axis_ranges: {1}\n" + \
-          "axis_ranges must be None, a [min, max] pair, or " + \
-          "an array-like giving the edge values of each pixel, " + \
-          "i.e. length must be length of axis + 1.".format(j, axis_ranges[j])
-
-        # Define function for converting pixel edges to pixel centers
-        edges_to_centers_nd = lambda axis_range: \
-          (axis_range[1:] - axis_range[:-1])/2 + axis_range[:-1]
+        def incompatible_axis_ranges_error_message(j):
+            return "Unrecognized format for {0}th entry in axis_ranges: {1}\n" + \
```

Comment:
Rather than using `+ \` you would be better off putting the whole thing in `()` and just splitting the string.